docs/mvvm/MigratingFromMvvmLight.md

```diff
@@ -0,0 +1,791 @@
+
+However, the `ObservableRecipient` exposes an `IsActive` property that also controls the call to these methods when it is set. 
+
+#### Set<T> ( string, ref T, T, bool )
+
+`Set<T>(string, ref T, T, bool)` does not have a like-for-like method signature replacement. 
+
+However, `SetProperty<T>(ref T, T, bool, string)` provides the same functionality with re-ordered parameters.
+
+```csharp
+// MvvmLight
+this.Set(nameof(this.MyProperty), ref this.myProperty, value, true);
+
+// Toolkit.Mvvm
+this.SetProperty(ref this.myProperty, value, true, nameof(this.MyProperty));
+```
+
+Note, the value and broadcast boolean parameters are not optional in the Toolkit's implementation and must be provided to use this method.
+
+Also, string parameter is not required if the method is being called from the property's setter as it is inferred from the caller member name.
+
+#### Set<T> ( ref T, T, bool, string )
+
+`Set<T>(ref T, T, bool, string)` has a renamed direct replacement, `SetProperty<T>(ref T, T, bool, string)`. 
+
+```csharp
+// MvvmLight
+this.Set(ref this.myProperty, value, true, nameof(this.MyProperty));
+
+// Toolkit.Mvvm
+this.SetProperty(ref this.myProperty, value, true, nameof(this.MyProperty));
+```
+
+Note, string parameter is not required if the method is being called from the property's setter as it is inferred from the caller member name.
+
+#### Set<T> ( Expression, ref T, T, bool )
+
+`Set<T>(Expression, ref T, T, bool)` does not have a direct replacement. 
+
+It is recommended for improved performance that you replace this with the Toolkit's `SetProperty<T>(ref T, T, bool, string)` using the `nameof` keyword instead.
+
+```csharp
+// MvvmLight
+this.Set<MyObject>(() => this.MyProperty, ref this.myProperty, value, true);
+
+// Toolkit.Mvvm
+this.SetProperty<MyObject>(ref this.myProperty, value, true, nameof(this.MyProperty));
+```
+
+Note, string parameter is not required if the method is being called from the property's setter as it is inferred from the caller member name.
+
+#### Broadcast<T> ( T, T, string )
+
+`Broadcast<T>(T, T, string)` has a direct replacement which doesn't require a rename. 
+
+```csharp
+// MvvmLight
+this.Broadcast<MyObject>(this.oldValue, this.newValue, nameof(this.MyProperty));
+
+// Toolkit.Mvvm
+this.Broadcast<MyObject>(this.oldValue, this.newValue, nameof(this.MyProperty));
+```
+
+Note, the message sent via the `Messenger` property when calling the `Broadcast` method has a direct replacement for `PropertyChangedMessage` within the Toolkit's framework. 
+
+Also, string parameter is not required if the method is being called from the property's setter as it is inferred from the caller member name.
+
+#### RaisePropertyChanged<T> ( string, T, T, bool )
+
+There is no direct replacement for the `RaisePropertyChanged<T>(string, T, T, bool)` method and any code using this should be altered or removed.
```

Comment:
Should we mention here that while there is no direct replacement, the same can be achieved with:
```csharp
OnPropertyChanged(nameof(MyProperty));
Broadcast(oldValue, newValue, nameof(MyProperty));
```
Or, without the need to do this manually, devs can of course also just call one of the `SetProperty` method to set additional dependent properties at anny point, and have that method take care of all this.

Reply:
On this one, I've gone with the first recommendation rather than the latter. The reason for this is the attempt to replicate the functionality. 

Although I hope it wasn't being used in this way, it's possible that this method was being called on properties which hadn't technically "changed". So calling `SetProperty` would lead to the broadcast not being called when doing the equality comparison on the values.